odoo/addons/base/models/ir_qweb.py

```diff
@@ -196,26 +1941,30 @@ def _compile_directive_call_assets(self, el, options, indent):
         if len(el):
             raise SyntaxError("t-call-assets cannot contain children nodes")
 
-        code = self._flushText(options, indent)
+        code = self._flush_text(options, indent)
+        code.append(self._indent("""t_call_assets_nodes = self._get_asset_nodes(%(xmlid)s, css=%(css)s, js=%(js)s, debug=values.get("debug"), async_load=%(async_load)s, defer_load=%(defer_load)s, lazy_load=%(lazy_load)s, media=%(media)s)""".strip() % {
+                'xmlid': repr(el.attrib.pop('t-call-assets')),
+                'css': self._compile_bool(el.attrib.pop('t-css', True)),
+                'js': self._compile_bool(el.attrib.pop('t-js', True)),
+                'async_load': self._compile_bool(el.attrib.pop('async_load', False)),
+                'defer_load': self._compile_bool(el.attrib.pop('defer_load', False)),
+                'lazy_load': self._compile_bool(el.attrib.pop('lazy_load', False)),
+                'media': repr(el.attrib.pop('media')) if el.get('media') else False,
+            }, indent))
```

Comment:
What about an f-string instead, and splitting the code on multiple lines?
```suggestion
        xmlid = el.attrib.pop('t-call-assets')
        css = self._compile_bool(el.attrib.pop('t-css', True))
        js = self._compile_bool(el.attrib.pop('t-js', True))
        async_load = self._compile_bool(el.attrib.pop('async_load', False))
        defer_load = self._compile_bool(el.attrib.pop('defer_load', False))
        lazy_load = self._compile_bool(el.attrib.pop('lazy_load', False))
        media = el.attrib.pop('media', False)
        code.append(self._indent(f"""
            t_call_assets_nodes = self._get_asset_nodes(
                {xmlid!r},
                css={css},
                js={js},
                debug=values.get("debug"),
                async_load={async_load},
                defer_load={defer_load},
                lazy_load={lazy_load},
                media={media!r},
            )
        """.strip(), indent))
```